src/services/notehub/models/NotehubRoutedEvent.ts

```diff
@@ -32,20 +32,35 @@ export interface NotehubRoutedEventLocationFields {
   tri_points?: number;
 }
 
-interface NotehubRoutedEvent
-  extends SessionMetadata,
-    NotehubRoutedEventLocationFields {
+interface NotehubRoutedEvent extends SessionMetadata, NotehubRoutedEventLocationFields {
+  app?: string; // notehub app/project uid
+  project?: {   // keeping this for backwards compatibility
+    id: string
+  }
```

Comment:
why? I don't think notehub is going back. If they did they would keep the `app` field so this gains nothing.